stan/math/opencl/kernels/helpers.hpp

```diff
@@ -11,7 +11,7 @@ namespace opencl_kernels {
 /*
  * Defines some helper macros for the kernels
```

Comment:
Please expand these comments with a little infor on what the helpers help at. Same for the thread block helpers. Something like "Helpers for readable matrix element access."